src/test/java/uk/gov/justice/digital/hmpps/csr/api/controllers/DetailResourceTest.kt

```diff
@@ -332,9 +333,26 @@ class DetailResourceTest : ResourceTest() {
         .headers(setAuthorisation(roles = ADMIN_ROLE))
         .exchange()
         .expectStatus().isOk
+        .expectBody<String>()
+        .consumeWith { c -> assertThat(c.responseBody!!.contains("Deleted 2 rows, time taken ")).isTrue() }
```

Comment:
slightly awkward but time taken varies

Reply:
would be better to do a `jsonPath` and then compare e.g. https://github.com/ministryofjustice/nomis-user-roles-api/blob/bfbbcc9f99152bb726f6b6e1d2fbf8ffb26d0bae/src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserRoleManagementResourceIntTest.kt#L197 with a `contains` or even https://github.com/ministryofjustice/nomis-user-roles-api/blob/bfbbcc9f99152bb726f6b6e1d2fbf8ffb26d0bae/src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserRoleManagementResourceIntTest.kt#L704